estate/models/__init__.py

```diff
@@ -0,0 +1 @@
+from . import estate_model
```

Comment:
```suggestion
from . import estate
```
No need to add '_model' as we already know it's a model. You can change the file name as well